ci-operator/config/openshift/hypershift/openshift-hypershift-release-4.16__periodics-mce.yaml

```
@@ -163,6 +163,13 @@ tests:
     env:
       MCE_VERSION: "2.6"
     workflow: hypershift-mce-ibmz-conformance
+- as: e2e-power-heterogeneous-x86-power-conformance
```

```suggestion
- as: e2e-x86-power-heterogeneous-conformance
```
Can we rename something like this?